packages/matchbox/src/components/Slider/Slider.js

```diff
@@ -151,64 +196,62 @@ function Slider(props) {
         window.removeEventListener('touchmove', handleTouchMove);
         window.removeEventListener('touchend', handleEnd);
       }
-    });
+    };
   }, [moving]);
 
-  const sliderClasses = classnames(
-    styles.Slider,
-    disabled && styles.Disabled
-  );
-
-  const tickMarkup = Object.keys(tickLocations).map((tick) => {
+  const tickMarkup = Object.keys(tickLocations).map(tick => {
     const { label, position } = tickLocations[tick];
     return (
-      <div
+      <StyledTick
         key={tick}
         style={{ left: position }}
-        className={classnames(styles.Tick, tick < sliderValue && styles.Included)}>
-        <div className={styles.TickLabel}>{label}</div>
-      </div>
+        disabled={disabled}
+        included={tick < sliderValue}
+      >
+        <StyledTickLabel>{label}</StyledTickLabel>
+      </StyledTick>
     );
   });
 
   return (
-    <div
-      className={sliderClasses}
-      data-id='slider-wrapper'
+    <StyledSlider
+      hasTicks={ticks}
+      disabled={disabled}
+      data-id="slider-wrapper"
       onTouchStart={disabled ? noop : handleTouchStart}
       onMouseDown={disabled ? noop : handleMouseDown}
       ref={sliderRef}
+      {...rest}
```

Comment:
Two things Ive found
- This outer wrapper is missing `data-id='slider-wrapper'`, because `...rest` is being spread here
- Test is finding two `data-id='slider-test'` passed from the story

I think we need to use that `pick` function on this object before passing it through so data-id doesnt get overwritten

Reply:
```js
// [data-id="slider-wrapper"]
CypressError: Timed out retrying: Expected to find element: '[data-id="slider-wrapper"]', but never found it.

// [data-id="slider-test"]
CypressError: cy.focus() can only be called on a single element. Your subject contained 2 elements.
```